operator/duties/scheduler.go

```diff
@@ -20,19 +22,34 @@ import (
 	"github.com/bloxapp/ssv/logging/fields"
 	"github.com/bloxapp/ssv/networkconfig"
 	"github.com/bloxapp/ssv/operator/duties/dutystore"
+	"github.com/bloxapp/ssv/operator/slotticker"
 	"github.com/bloxapp/ssv/protocol/v2/types"
 )
 
 //go:generate mockgen -package=mocks -destination=./mocks/scheduler.go -source=./scheduler.go
 
+var slotDelayHistogram = promauto.NewHistogram(prometheus.HistogramOpts{
+	Name:    "slot_ticker_delay_milliseconds",
+	Help:    "The delay in milliseconds of the slot ticker",
+	Buckets: []float64{5, 10, 20, 100, 500, 5000}, // Buckets in milliseconds. Adjust as per your needs.
+})
+
+func init() {
```

Comment:
I think we need to add new metrics to `MetricsReporter` and pass them instead of creating them in `init` for new packages

Reply:
okay, I see it's still something to be discussed, so let's not make the PR depend on this